Applications/Games/League of Legends/Online/script.js

```diff
@@ -94,12 +79,17 @@ var installerImplementation = {
             })
             .category("Games")
             .wineDistribution("staging")
-            .wineVersion(LATEST_STAGING_VERSION)
+            .wineVersion(LATEST_STAGING_VERSION) // TODO: Gallium9 patchset
             .preInstall(function (wine /*, wizard*/) {
                 wine.windowsVersion("winxp");
                 wine.d3dx9();
-                wine.overrideDLL().set("native, builtin", ["atl120", "msvcp120", "msvcr120", "vcomp120", "msvcp140"]).do();
+                // wine.overrideDLL().set("native, builtin", ["atl120", "msvcp120", "msvcr120", "vcomp120", "msvcp140"]).do(); // Obsolete?
                 wine.enableCSMT();
+                wine.disableGLSL(); // Needs approval + recommended by WineHQ
+                wine.corefonts(); // Needs approval + mandatory for fonts
+                wine.vcrun2008(); // Needs approval
+                wine.vcrun2017(); // Needs approval
+                wine.adobeair(); // Needs approval + mandatory for launcher
```

Comment:
I do not think there is a verb for that currently.

Reply:
You can try to do it yourself. Basically you just have to download `https://airdownload.adobe.com/air/win/download/latest/AdobeAIRInstaller.exe` and launch it.

Reply:
@ImperatorS79 I've commited the verb based on available informations.